gulp/tasks/test.js

```diff
@@ -95,6 +94,18 @@ function _getProtractorParams() {
   return params;
 }
 
+function _getWCAGParams() {
```

Comment:
Yes! Added in https://github.com/cfpb/cfgov-refresh/commit/4550c8f0ea9e3a46e07fa875445a04de35a54205